README.md

```diff
@@ -70,6 +70,20 @@ The language breakdown analysis tab also displays a table showing the Accepted P
 
 4. **Total Active Copilot Chat Users:** a bar chart that illustrates the total number of users who have actively interacted with Copilot over the past 28 days.
 
+## Seat Analysis 
+
+![image](https://github.com/DevOps-zhuang/copilot-metrics-viewer/assets/54096296/d1fa9d1d-4fab-4e87-84ba-7be189dd4dd0)
```

Comment:
We might need to update this image once the changes have been implemented 🙏🏼

Reply:
Issue created: [README Seat Analysis image update](https://github.com/github-copilot-resources/copilot-metrics-viewer/issues/35)